client/android/.gitignore

```diff
@@ -0,0 +1,13 @@
+gradle-wrapper.jar
```

Comment:
I think we should remove the folders we don't need for now to avoid confusing contributors.

So remove android / ios / linux / macos / windows  folders.

Reply:
The term in programmers culture is YAGNI (You ain't gonna need it) to avoid putting things in a project while you're not sure it's useful.

Reply:
cool!